ci-lua

```diff
@@ -1,25 +1,43 @@
-#!/bin/sh
+#!/bin/bash
 
 . ./config
 
-rm -rf "$TEST_OUT_DIR/lua"
-mkdir -p "$TEST_OUT_DIR/lua"
+ALLOWED_LUA_VERSIONS=("5.1" "5.2" "5.3")
+CURRENT_LUA_VERSION=""
+
+# try to use $VARIETY from env
+for allowed_version in "${ALLOWED_LUA_VERSIONS[@]}"; do
+  if [[ "${allowed_version}" = "${VARIETY}" ]]; then
+    CURRENT_LUA_VERSION="${VARIETY}"
+  fi
+done
+
+if [ -z "${CURRENT_LUA_VERSION}" ]; then
+  # fallback to version 5.1
+  CURRENT_LUA_VERSION="5.1"
+fi
+
+LUA_OUT_DIR="${TEST_OUT_DIR}/lua${CURRENT_LUA_VERSION}"
```

Comment:
I don't think that different Lua versions should have separate directories. The current approach is that one language goes only into one directory, the language version doesn't matter. Using a special branch for every target in the [`ci_artifacts` repo](https://github.com/kaitai-io/ci_artifacts/) ensures that there are no conflicts.

See branches [`python/2.7`](https://github.com/kaitai-io/ci_artifacts/tree/python/2.7) and [`python/3.4`](https://github.com/kaitai-io/ci_artifacts/tree/python/3.4) for instance, all test artifacts go to directory `test_out/python` in both cases, ~**not** `test_out/python2`~ or something like that.

Reply:
Well, maybe I'm confused by ci-python, ci-python3, ci-python3 scripts, that use different directories. However it seems those are not run in Travis CI, only the ci-python one in differently configured environment.
I've reverted, and hopefuly the https://github.com/kaitai-io/kaitai_ci_ui/pull/8 will suffice to differentiate test targets